submissions/SkirnevskyiAleksandr/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <!--header-->
+    <header class="container">
+      <!--header-menu-->
+      <nav class="menu">
+        <ul class="menu">
+          <li class="menu-item"><a class="menu-item" href="#">Mail</a></li>
+          <li class="menu-item"><a class="menu-item" href="#">Images</a></li>
+          <li class="menu-item">
+            <!--dropdown-btn-->
+            <input type="checkbox" id="drop-down-btn" class="drop-down-input" />
+            <label class="drop-down-input" for="drop-down-btn"
+              ><img
+                class="picture"
+                src="./icons/popup-button.png"
+                alt="popup-button"
+            /></label>
+            <!--dropdown-menu-->
+            <nav class="dropdown-container">
+              <ul class="drop-down-menu">
```

Comment:
Try to write class names or JS variables names in one style, It easier to read it in the future)